CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.49.8
+
+* Fix a bug where `require('sass')` was broken on Node 17 due to Dart's JS
+  interop.
```

Comment:
This kind of makes it sound like loading the module at all was impossible. Maybe instead, "Fix a bug where inspecting the Sass module in the Node.js console crashed on Node 17."

Also, put this under a `## JS API` heading.